appengine/gophers/gophers-2/main.go

```diff
@@ -21,6 +16,4 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 		http.Redirect(w, r, "/", http.StatusFound)
 		return
 	}
-
-	indexTemplate.Execute(w, nil)
```

Comment:
This means the page will be blank. We still want to render the template, even though there aren't any parameters.

It's the same as writing out the index file.

Reply:
Or is this because you added a static handler for the index page?

Reply:
static handler! WDYT?

Reply:
Sounds good.

This looks OK.